### ISSUE TITLE
New command `Create New Terminal in Editor Area (With Profile)`

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#41473]
When opening a new terminal during dev, I usually want this to open in the editor and therefore use the command `Terminal: Create New Terminal in Editor Area`.

This is mostly fine but I sometimes have to open a terminal with a profile != my default one. This is only possible with `Terminal: Create New Terminal (With Profile)` which doesn't open the terminal in my editor but in the "view" (a panel at the bottom in my case).

Whilst this is okish, I'd usually prefer to also open this in the editor.

I realize that setting `terminal.integrated.defaultLocation` to `editor` would solve this but this also changes all tasks to run in the editor which is a no-go for me and there's no way (that I know of) to force my tasks to keep running in the "view" with this setting.

- In short

An additional `Create New Terminal in Editor Area (With Profile)` to the existing `Terminal: Create New Terminal`, `Terminal: Create New Terminal (With Profile)` & `Terminal: Create New Terminal in Editor Area` would really make sense in general and would be really helpful to my personal workflows.
